src/Neo.CLI/CLI/MainService.Blockchain.cs

```diff
@@ -257,7 +257,7 @@ public void OnShowContractCommand(string nameOrHash)
                 ConsoleHelper.Info("", "            Compiler: ", $"{contract.Nef.Compiler}");
                 ConsoleHelper.Info("", "          SourceCode: ", $"{contract.Nef.Source}");
                 ConsoleHelper.Info("", "              Trusts: ", $"[{string.Join(", ", contract.Manifest.Trusts.Select(s => s.ToJson()?.GetString()))}]");
-                if (contract.Manifest.Extra is null)
+                if (contract.Manifest.Extra is not null)
```

Comment:
This was a bug

Reply:
someone must of changed the code, i don't write code like that.